.openzeppelin/project.json

```
@@ -3,8 +3,8 @@
   "name": "wildcards-contracts",
   "version": "0.1.0",
   "contracts": {
-    "ERC721Patronage": "ERC721Patronage_v1",
-    "WildcardSteward": "WildcardSteward_v2",
+    "ERC721Patronage": "ERC721Patronage_v0",
+    "WildcardSteward": "WildcardSteward_v0",
     "MintManager": "MintManager_v2"
```

That said, I might as well reverse this

```suggestion
    "ERC721Patronage": "ERC721Patronage_v1",
    "WildcardSteward": "WildcardSteward_v2",
    "MintManager": "MintManager_v2"
```